docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.6'
+
+volumes:
+  pg_marketplace:
+
+services:
+  postgres:
+    image: postgres:latest
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - pg_marketplace:/var/lib/postgresql/data
+  app:
+    build: .
+    ports:
+      - "8080:8080"
+    depends_on:
+      - postgres
+    env_file:
+      - .env
```

Comment:
can we have a readme file for this one please?

Reply:
we can have a docker file and compose file in `docker` folder 
and readme file would have these instructions: 
- build image with Dockerfile:
    ```shell
    docker build --tag food-backend --file ./docker/Dockerfile .
    ```
- compose and start back-end and db container:
    ```shell
    docker-compose --file ./docker/docker-compose.yml up -d
    ```